packages/lib-classifier/src/store/SubjectViewerStore.spec.js

```diff
@@ -1,23 +1,48 @@
+import asyncStates from '@zooniverse/async-states'
 import SubjectViewerStore from './SubjectViewerStore'
 
-let subjectViewerStore
-
-describe('Model > SubjectViewerStore', function () {
-  before(function () {
-    subjectViewerStore = SubjectViewerStore.create()
-  })
-
+describe.only('Model > SubjectViewerStore', function () {
```

Comment:
prob intentional part of this process of merging mini-PRs into main related PR, but just noting `.only`

Reply:
Yep, each had the `only` purposely left in to demonstrate that the tests for the store were passing and the branch they merge into is for testing that they all pass.